weblate/trans/tasks.py

```diff
@@ -17,7 +17,8 @@
 from django.contrib.staticfiles.storage import staticfiles_storage
 from django.core.cache import cache
 from django.db import IntegrityError, transaction
-from django.db.models import Count, F
+from django.db.models import Count, DateTimeField, ExpressionWrapper, F, Value
```

Comment:
Add import for type annotation:

```suggestion
from django.db.models import Count, DateTimeField, Expression, ExpressionWrapper, F, Value
```